.github/workflows/CI.yml

```diff
@@ -44,6 +44,11 @@ jobs:
           cd docs
```

Comment:
The tests should run correctly with local sub-packages.
The reason is that, at this line, we enter the `docs` folder first. And under the `docs` folder, the `Manifest.toml` file is committed into the repo with sub-packages correctly specified.

I know it is a bit tricky 😢 The reason I do so is to avoid maintaining dependencies in different places.

Reply:
Ah I missed that completely. It is a little weird, but I understand why.